sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java

```diff
@@ -95,4 +96,15 @@ public SdkLoggerProviderBuilder setClock(Clock clock) {
   public SdkLoggerProvider build() {
     return new SdkLoggerProvider(resource, logLimitsSupplier, logRecordProcessors, clock);
   }
+
+  /**
+   * Merge custom resource object with the current resource object.
+   *
+   * @param resource {@link Resource} object to merge with current.
+   */
+  public SdkLoggerProviderBuilder mergeResource(Resource resource) {
```

Comment:
maybe `addResource` - merge sounds like the method would return the merged object and I have to do something with it.

Reply:
Done. 
Thanks for the suggestion.